### PR TITLE
[tag, input-tags] added accessibility

### PR DESCRIPTION
**Pull request by ekrasnopolskaia**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ekrasnopolskaia** [comment]
InputTag doesn't have a11y tab, should we make up something?

File: website/docs/components/input-tags/examples/basic.jsx [hunk 1]
```diff
@@ -50,14 +56,20 @@ const Demo = () => {
             editable
             data-id={idx}
             onClick={handleEditTag}
+            onKeyDown={handleKey}
           >
             <InputTags.Tag.Text>{tag}</InputTags.Tag.Text>
             <InputTags.Tag.Close onClick={handleCloseTag} />
```
Comment (lsroman) [inline website/docs/components/input-tags/examples/basic.jsx:62]: `onClick={handleCloseTag}` why is it here? It doesn't do anything

File: website/docs/components/input-tags/examples/basic.jsx [hunk 2]
```diff
@@ -50,14 +56,20 @@ const Demo = () => {
             editable
             data-id={idx}
             onClick={handleEditTag}
+            onKeyDown={handleKey}
```
Comment (lsroman) [inline website/docs/components/input-tags/examples/basic.jsx:59]: Can do as with icons? What if was call a click then a keydown is called

File: website/docs/components/input-tags/examples/basic.jsx [hunk 3]
```diff
@@ -50,14 +56,20 @@ const Demo = () => {
...
           </Tooltip.Trigger>
           <Tooltip.Popper>tag</Tooltip.Popper>
         </Tooltip>
       ))}
-      <InputTags.Value value={value} onChange={updateValue} onBlur={handleBlurInput} />
+      <InputTags.Value
+        value={value}
+        onChange={updateValue}
+        onBlur={handleBlurInput}
+        aria-label="input with tags"
```
Comment (lsroman) [inline website/docs/components/input-tags/examples/basic.jsx:71]: Could you synchronize with ux specialists and understand what is more important here, a placeholder or a label?

**Review by lsroman — CHANGES_REQUESTED** [review]
I've added a few changes. Let's discuss these changes in a voice so it will be faster?

File: website/docs/components/input-tags/examples/basic.jsx (hunk 3 above)
Comment (ekrasnopolskaia) [reply website/docs/components/input-tags/examples/basic.jsx:71]: Julia said that label is more important

File: website/docs/components/input-tags/examples/basic.jsx (hunk 1 above)
Comment (phytonmk) [reply website/docs/components/input-tags/examples/basic.jsx:62]: Same question

Comment (ekrasnopolskaia) [reply website/docs/components/input-tags/examples/basic.jsx:62]: Why? It deletes the tag when you click the cross. Clicking on tag sends tag to edit, but clicking on the cross deletes it

**Review by lsroman — APPROVED** [review]
😍 thanks for your work!

**Pull request merged**